task_steps/views.py

```diff
@@ -0,0 +1,32 @@
+from rest_framework import generics, status
+from .models import TaskStep
+from .serializers import TaskStepSerializer
+from rest_framework.response import Response
+
+
+class TaskStepListCreateView(generics.ListCreateAPIView):
+    queryset = TaskStep.objects.all()
+    serializer_class = TaskStepSerializer
```

Comment:
Because of Natania's suggestion, This is not important, also this implementation is probably incorrect. You have to filter it based on the id task as well.

Reply:
I agree with this approach as there is no need for the List on the APIView hence you can just use the CreateAPIView.

Reply:
Hi catherine this is your code, I'll just change it immediately if that's okay.